--- a/libs/maildrop/maildrop.C
+++ b/libs/maildrop/maildrop.C
@@ -99,7 +99,7 @@ catch (const char *p)
 		std::cerr << argv[0] << ": " << p << "\n"
 			  << std::flush;
 #if SYSLOG_LOGGING
-		syslog(LOG_INFO, "%s", p);
+		syslog(LOG_INFO, "%s", p);
 #endif
 		cleanup();
 		return (EX_TEMPFAIL);
@@ -109,7 +109,7 @@ catch (char *p)
 		std::cerr << argv[0] << ": " << p << "\n"
 			  << std::flush;
 #if SYSLOG_LOGGING
-		syslog(LOG_INFO, "%s", p);
+		syslog(LOG_INFO, "%s", p);
 #endif
 		cleanup();
 		return (EX_TEMPFAIL);
