--- maildrop/maildrop.C	2011-04-04 17:58:58.000000000 +0200
+++ maildrop/maildrop.C.oden	2011-10-07 13:57:40.632808723 +0200
@@ -80,7 +80,7 @@ int	n;
 	{
 		merr << argv[0] << ": " << p << "\n";
 #if SYSLOG_LOGGING
-		syslog(LOG_INFO, p);
+		syslog(LOG_INFO, "%", p);
 #endif
 		cleanup();
 		return (EX_TEMPFAIL);
@@ -90,7 +90,7 @@ int	n;
 	{
 		merr << argv[0] << ": " << p << "\n";
 #if SYSLOG_LOGGING
-		syslog(LOG_INFO, p);
+		syslog(LOG_INFO, "%", p);
 #endif
 		cleanup();
 		return (EX_TEMPFAIL);
